### DISCUSSION TITLE
Adding a simple Title-Sequence  (Headline) to video

### DISCUSSION THREAD

**Discussion by McEilif** [mifi/lossless-cut#1808] [Ideas]
Hi Mikael, Again winter and time to clean up in the past years video-recordings from all our Mobile-phones...

In our family we enjoy watching the videos we have made from whereever together in the livingroom on a TV. Before that we have used your lovely software to chub up and shorten the collection of videos and then merged them into one "Movie". Imagine fx "Our homevideos January 2023", or "The time Doggy got Puppies 2014", "The wedding of XXX & XXX - 2021".

Now, on the TV - picking the Videos from our NAS - we sometimes just let it run all the videos from a folder on the NAS, which is arranged after year and months in our case. And therefore I add a Title-sequence in the beginning of each "Movie", normally between 5 and 8 seconds, depending how much information we write on it.
And here is the Problem, which I guess is my lack of knowledge about frames and videoediting.
Even if I try importing the merged video into KdenLive just for getting same framerate and format, then create just the Title-sequence, which I then export as same fileformat as the Movie - then again merge Title+Movie in LLC to get a complete Movie with fx 5 seconds of black screen with information of what we are going to watch... 
Unfortunately that sometimes doesn't work as intended - Even if my 5 seconds Intro has 125 frames for 5 seconds, it sometimes block the 126th and all there after frames from viewing - the audio is there, but else just the title... and in some cases then after a while (maybe after 125seconds) the video finally comes but in slow-motion with soundtrack ended...

Now the idea (whish) - It might be out of the Concept for LLC, but please concider... 
Add a simple "Title-Clip Creation" fx under the "Tools" menu... Selection of backgroundcolor, textcolor, some few standard fonts, and length in seconds, that we could place in front, in the end, (and maybe in between cuts - though that would probably be a lot more work, due to Audio tracks and such).

In my case I have to re-render a complete video, just because of the need of putting some texts in here and there... Kdenlive is not that good as Handbrake for rendering, so I normally cut originals with LLC, then create raw movie in Kdenlive in highest (nearly) lossless Quality, then again render to acceptable Quality-Filesize in Handbrake...

Thanks for reading this far, and a lovely software (have said that before) and Merry Christmas!

**Comment by mifi** [comment]
Hi.

> Now, on the TV - picking the Videos from our NAS - we sometimes just let it run all the videos from a folder on the NAS, which is arranged after year and months in our case. And therefore I add a Title-sequence in the beginning of each "Movie", normally between 5 and 8 seconds, depending how much information we write on it. And here is the Problem, which I guess is my lack of knowledge about frames and videoediting. Even if I try importing the merged video into KdenLive just for getting same framerate and format, then create just the Title-sequence, which I then export as same fileformat as the Movie - then again merge Title+Movie in LLC to get a complete Movie with fx 5 seconds of black screen with information of what we are going to watch... Unfortunately that sometimes doesn't work as intended - Even if my 5 seconds Intro has 125 frames for 5 seconds, it sometimes block the 126th and all there after frames from viewing - the audio is there, but else just the title... and in some cases then after a while (maybe after 125seconds) the video finally comes but in slow-motion with soundtrack ended...

I'm not sure I understand exactly. Did you try the troubleshooting steps and see if that helps? in particular the information about keyframes and accurate cutting.

> Add a simple "Title-Clip Creation" fx under the "Tools" menu... Selection of backgroundcolor, textcolor, some few standard fonts, and length in seconds, that we could place in front, in the end, (and maybe in between cuts - though that would probably be a lot more work, due to Audio tracks and such).

I believe what you're asking is point 1 in the FAQ: https://github.com/mifi/lossless-cut/blob/master/issues.md#faq - lossy editing?

**Reply by McEilif** [reply]
regarding to the FAQ it says:

> Can LosslessCut crop, resize, stretch, mirror, overlay text/images, watermark, blur, redact, re-encode, create GIF, slideshow, burn subtitles, color grading, fade/transition between video clips, fade/combine/mix/merge audio tracks or change audio volume?
> 
>     No, these are all lossy operations (meaning you have to re-encode the file), but in the future I may start to implement such features. 

Well I didn't see this page.
I didn't write this as a bug or error message - simply a whish, and for you to better understand the whish/needs I have, I wrote the way we use TV to watch a collections of homevideos - it simply goes about the whish of letting LLC have an option, fx as a tool, to create a 5-10 seconds "Frozen Title" picture, where user can add some text on a colored or maybe a jpg-background", that LLC then encode and add to the video or movie, whatever the project is. This is in my opinion very useful when streaming or playing from NAS or USB a homevideo.
We may also not forget, that older people or like me, not buys new TVs everytime a new model comes out with new features. Fx neither my mother-in-law or my father (They are both around 80) will never buy a new TV again as long as it works. And they enjoy the principle of "plug-and-play", and for that reason I started converting all the old Analog Videos and Film-rolls for them to watch the old stuff.

Here is a small example to see what I mean:
https://github.com/mifi/lossless-cut/assets/118133964/2a19c02f-a39b-4e11-81e4-3c4fe1903501

**Comment by McEilif** [comment]
> Now the idea (whish) - It might be out of the Concept for LLC, but please concider... Add a simple "Title-Clip Creation" fx under the "Tools" menu... Selection of backgroundcolor, textcolor, some few standard fonts, and length in seconds, that we could place in front, in the end, (and maybe in between cuts - though that would probably be a lot more work, due to Audio tracks and such).

Sorry if I confused you with too much text - this was my idea/whish which I guess is mentioned in the above linked F.A.Q.

**Reply by mifi** [reply]
I think it's kind of similar to adding "pause" between segments, #255
